libavcodec/x86/vvc/vvcdsp_init.c

```diff
@@ -233,10 +247,12 @@ void ff_vvc_dsp_init_x86(VVCDSPContext *const c, const int bd)
         if (EXTERNAL_AVX2_FAST(cpu_flags)) {
             MC_LINKS_AVX2(12);
             MC_LINKS_16BPC_AVX2(12);
+            SAD_INIT();
         }
     }
 
     if (EXTERNAL_AVX2(cpu_flags)) {
+        SAD_INIT();
```

Comment:
this not needed since you init them in bd 8, 10, 12.

Reply:
Sounds good, I guess one thing I'm not sure about is what the difference between EXTERNAL_AVX2 and EXTERNAL_AVX2_FAST ?

Reply:
Not clear, maybe some machines are not fast.
If we do not have 8, 10, 12 bit versions of sad, maybe we can init it only once